projects/BFDR.CLI/Program.cs

```diff
@@ -110,7 +195,255 @@ static int Main(string[] args)
+                BirthRecord indr = new BirthRecord(File.ReadAllText(args[1]));
+                BirthRecord outdr = new BirthRecord();
+                List<PropertyInfo> properties = typeof(BirthRecord).GetProperties().ToList();
+                foreach (PropertyInfo property in properties)
+                {
+                    if (property.GetCustomAttribute<Property>() != null)
+                    {
+                        property.SetValue(outdr, property.GetValue(indr));
+                    }
+                }
+                Console.WriteLine(outdr.ToJSON());
+                return 0;
+            }
+            else if (args.Length == 2 && args[0] == "roundtrip-ije")
+            {
+                // Console.WriteLine("Converting FHIR to IJE...\n");
+                BirthRecord b = new BirthRecord(File.ReadAllText(args[1]));
+                IJENatality ije1, ije2, ije3;
+                try
+                {
+                    ije1 = new IJENatality(b);
+                    ije2 = new IJENatality(ije1.ToString());
+                    ije3 = new IJENatality(new BirthRecord(ije2.ToRecord().ToXML()));
+                }
+                catch (Exception e)
+                {
+                    Console.Error.WriteLine(e.Message);
+                    return (1);
+                }
+
+                int issues = 0;
+                int total = 0;
+                foreach (PropertyInfo property in typeof(IJENatality).GetProperties())
+                {
+                    string val1 = Convert.ToString(property.GetValue(ije1, null));
+                    string val2 = Convert.ToString(property.GetValue(ije2, null));
+                    string val3 = Convert.ToString(property.GetValue(ije3, null));
+
+                    IJEField info = property.GetCustomAttribute<IJEField>();
+
+                    if (val1.ToUpper() != val2.ToUpper() || val1.ToUpper() != val3.ToUpper() || val2.ToUpper() != val3.ToUpper())
+                    {
+                        issues++;
+                        Console.WriteLine($"[***** MISMATCH *****]\t{info.Name}: {info.Contents} \t\t\"{val1}\" != \"{val2}\" != \"{val3}\"");
+                    }
+                    total++;
+                }
+                Console.WriteLine($"\n{issues} issues out of {total} total fields.");
+                return issues;
+            }
+            else if (args.Length == 2 && args[0] == "roundtrip-all")
+            {
+                BirthRecord b1 = new BirthRecord(File.ReadAllText(args[1]));
+                BirthRecord b2 = new BirthRecord(b1.ToJSON());
+                BirthRecord b3 = new BirthRecord();
+                List<PropertyInfo> properties = typeof(BirthRecord).GetProperties().ToList();
+                // HashSet<string> skipPropertyNames = new HashSet<string>() { "CausesOfDeath", "AgeAtDeathYears", "AgeAtDeathMonths", "AgeAtDeathDays", "AgeAtDeathHours", "AgeAtDeathMinutes" };
```

Comment:
either remove the comment and if statement below, or if we want to keep it in case we need it later, just make the HashSet empty to clean up references to VRDR